index.html

```diff
@@ -52,6 +54,8 @@
       </div>
     </nav>
 
+    <!-- Adding a seatchbar, because they're cool! -->
```

Comment:
Noticed a typo on line 57. Seatchbar should be searchbar. 
Adding the searchbar is a great idea-- they really are cool!

Reply:
Thanks @georginajuarez fixed the typo!